### PR TITLE
Add the Home page with layout according to Figma design

### PR DESCRIPTION
**Pull request by NadiiaLashtun**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
-import './App.css';
-import Header from './Header';
-
-import UserData from './components/UserData';
+/*import "./App.css";*/
```
Comment (mnichols08) [inline src/App.jsx]: At some point we will want to remove this line but it's fine to merge with it for now.

File: src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+import Header from "./components/Header";
+import MainSection from "./components/MainSection";
+import UserData from "./components/UserData";
 
 function App() {
   return (
     <>
       <Header />
+      <MainSection />
       <UserData />
```
Comment (mnichols08) [inline src/App.jsx]: This line indicates it is possible that this commit is behind development. I do believe that `<UserData / >` component was removed in a previous PR.
It shouldn't matter but in the future its best to type `git fetch origin` `git pull` then `git merge origin/development` to merge your local branch with the most up-to-date code

File: src/App.jsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
-import './App.css';
-import Header from './Header';
-
-import UserData from './components/UserData';
+/*import "./App.css";*/
+import Header from "./components/Header";
+import MainSection from "./components/MainSection";
+import UserData from "./components/UserData";
```
Comment (mnichols08) [inline src/App.jsx]: I mention this in the next comment. But this was removed in a previous PR

File: src/components/Header.jsx
Comment (mnichols08) [inline src/components/Header.jsx:1]: I am a big fan of creating a new component for this, great job

File: src/components/MainSection.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Wrapper from "./Wrapper";
+import Arrow from "/src/img/arrow.png";
+import CircleButton from "/src/img/circle-button.svg";
+
+function MainSection() {
+  return (
+    <section className="bg-[url('/src/img/main-section-bg.jpg')] bg-cover bg-center w-full h-screen border-y border-secondary-color">
```
Comment (mnichols08) [inline src/components/MainSection.jsx]: It may be more 'semantic' to use a `<main>` element here

File: src/components/MobileMenu.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
```
Comment (mnichols08) [inline src/components/MobileMenu.jsx:1]: I don't think that we have to import React into a functional component

File: src/components/NavItem.jsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (mnichols08) [inline src/components/NavItem.jsx:1]: same here with importing React. But still a huge fan of separate components

File: tailwind.config.js
Comment (mnichols08) [inline tailwind.config.js:1]: Thanks for configuring tailwind here =) looking good for me

**Review by mnichols08 — APPROVED** [review]
Great work @NadiiaLashtun! Thanks for taking on this task.


We have some conflicting files to resolve. If you like I can help with resolving conflicts.

In the future this can be avoided by typing 
```
git fetch origin
git merge origin/development
```
 into you console before beginning to code each day (this updates your local branch with the development branch on Github)

File: src/App.jsx (hunk 1 above)
Comment (NadiiaLashtun) [reply src/App.jsx]: I wasn't sure if I could delete this file due to the recent situation, so I left it as a comment

File: src/App.jsx (hunk 2 above)
Comment (NadiiaLashtun) [reply src/App.jsx]: Thank you for your advice. I actually forgot to merge the last changes before creating the PR.

File: src/components/MainSection.jsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+              Color Compass
+            </h1>
+            <p className="w-2/3 md:w-[460px]">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus
+              mattis tortor ut tortor cursus, ac maximus tortor.
+            </p>
+          </div>
+          <a
+            href="#"
+            className="md:grow-0 hover:animate-arrowRotate origin-left animate-pulse"
```
Comment (mnichols08) [inline src/components/MainSection.jsx]: I love the animations and the hover effect! It's an easter egg I didn't even notice at first but great touch =)

Comment (KseniiaRiabova) [reply src/components/MainSection.jsx]: Love the animation!

**Review by KseniiaRiabova — APPROVED** [review]
Good Job, it looks even better than I could imagine

**Review by frosty8104 — APPROVED** [review]

**Pull request merged**